src/main/resources/langs/be/admin.json

```diff
@@ -73,7 +73,7 @@
   "server_restrictions": "Абмежавання",
   "server_restrictions_none": "На гэты сервер не накладзена ніякіх абмежаванняў.",
   "server_restrictions_suspended": "Федэрацыя з гэтым серверам заблакаваная. Яго карыстальнікі не могуць ўзаемадзейнічаць з вашымі і наадварот.",
-  "server_restrictions_change": "Змяніць абмежаванні…",
+  "server_restrictions_change": "Змяніць абмежаванні...",
```

Comment:
The wrong ellipsis are back 😱

Reply:
Why does it keep messing with Russian translations, I don't understand. And why do integrations break themselves and need two buttons clicked to start working again. Different buttons each time, too.